### PR TITLE
feat1: Basic Routing and Header Component

### PR DESCRIPTION
**Pull request by SinchanaMS**
Features Added:
1. Basic Routing
2. Header Component and Styles
3. Mockman Integration

Files to Check:
1. Router.jsx
2. Header.jsx
3. App.js

Preview of the Header Component:
image

### CODE REVIEW

File: src/router/Router.jsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        <Routes>
+            <Route path="/" element={<LandingPage/>}/>
+            <Route path="/products" element={<ProductsList/>}/>
+            <Route path="/cart" element={<Cart/>}/>
+            <Route path="/wishlist" element={<Wishlist/>}/>
+            <Route path="/login" element={<Login/>}/>
+            <Route path="/signup" element={<SignUp/>}/>
+            <Route path="/api" element={<MockAPI />} />
+            <Route path="*" element={<PageNotFound />} />
+        </Routes>
```
Comment (aiman-ansari) [inline src/router/Router.jsx:17]: Great work as you created a separate component for Routes

**Review by aiman-ansari — APPROVED** [review]
- You can put your assest folder inside the  public folder.
- Great folder structure.
- Also you can add a separate index.js file inside the pages folder .
- Looks good to merge :)

File: src/components/Header/Header.css [hunk 2]
```diff
@@ -0,0 +1,51 @@
+@import "../../App.css";
+
+.header {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
```
Comment (srejitk) [inline src/components/Header/Header.css:6]: You can add this as a utility in your app.css for reusabilty

File: src/App.css [hunk 3]
```diff
@@ -1,51 +1,12 @@
...
+@import url('https://fonts.googleapis.com/css2?family=Aleo:ital,wght@0,300;0,400;0,700;1,400&family=Andika+New+Basic:ital,wght@1,700&family=Comforter&family=Montserrat:ital,wght@1,800&family=Open+Sans:wght@300;500&family=Roboto:wght@100;700&display=swap');
+@import url("https://clay-ui.netlify.app/stylesheets/index.css");
+
+:root {
+  --HEADER-BG: #fcfaf7;
+  --SEARCH-BAR-BG: #f0ecec;
+  --CATEGORIES-BG: #f5ebeb;
+  --CAROUSEL-BG: rgb(240, 232, 227);
+  --CAROUSEL-TEXT-BG: rgb(240, 232, 227);;
+  --NEW-ARRIVALS-BG: #ebdcdc;
```
Comment (srejitk) [inline src/App.css]: Try to stick to a single color code, either RGB or Hexcodes. 
Good Practice of naming convention.

**Review by srejitk — APPROVED** [review]
Looks clean and good. Have suggested a few changes.

File: src/components/Header/Header.jsx [hunk 4]
```diff
@@ -0,0 +1,33 @@
+import { Link } from 'react-router-dom'
+import mainLogo from "../../assets/mainLogo.png"
+import "./Header.css"
+
+export default function Header() {
+  return (
+    <header className="header shadow">
+        <Link to="/">
```
Comment (m26unkwn) [inline src/components/Header/Header.jsx:7]: You can add NavLink here for navigation.

**Review by m26unkwn — APPROVED** [review]

File: src/App.js [hunk 5]
```diff
@@ -1,41 +1,12 @@
...
-            target="_blank"
-            rel="noreferrer"
-          >
-            Contribute
-          </a>
-        </div>
-      </header>
+    <div>
+      <Header/>
+      <Router/>
```
Comment (tarunsankhla) [inline src/App.js:9]: Creating a file just for router is really  good approach,  I will do the same

File: src/App.css [hunk 6]
```diff
@@ -1,51 +1,12 @@
...
+@import url("https://fonts.googleapis.com/icon?family=Material+Icons");
+@import url('https://fonts.googleapis.com/css2?family=Aleo:ital,wght@0,300;0,400;0,700;1,400&family=Andika+New+Basic:ital,wght@1,700&family=Comforter&family=Montserrat:ital,wght@1,800&family=Open+Sans:wght@300;500&family=Roboto:wght@100;700&display=swap');
+@import url("https://clay-ui.netlify.app/stylesheets/index.css");
+
+:root {
+  --HEADER-BG: #fcfaf7;
+  --SEARCH-BAR-BG: #f0ecec;
+  --CATEGORIES-BG: #f5ebeb;
+  --CAROUSEL-BG: rgb(240, 232, 227);
+  --CAROUSEL-TEXT-BG: rgb(240, 232, 227);;
```
Comment (tarunsankhla) [inline src/App.css]: 2 semicolons

File: public/index.html [hunk 7]
```diff
@@ -10,6 +10,10 @@
       content="Web site created using create-react-app"
     />
     <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
+    <link
+  rel="stylesheet"
```
Comment (tarunsankhla) [inline public/index.html]: Intendation

**Review by tarunsankhla — APPROVED** [review]
Over all it is great

File: src/router/Router.jsx (hunk 1 above)
Comment (SinchanaMS) [reply src/router/Router.jsx:17]: Thank you for the review! I have made the suggested changes.

File: src/App.css (hunk 3 above)
Comment (SinchanaMS) [reply src/App.css]: Thank you for pointing it out, I'd missed it. Changed it to Hexcode.

File: src/App.css (hunk 6 above)
Comment (SinchanaMS) [reply src/App.css]: Corrected it, thank you for pointing it out.

File: src/components/Header/Header.jsx (hunk 4 above)
Comment (SinchanaMS) [reply src/components/Header/Header.jsx:7]: I'll check that out, thank you!

File: src/components/Header/Header.css (hunk 2 above)
Comment (SinchanaMS) [reply src/components/Header/Header.css:6]: I'll check that out, thank you!

**Comment by SinchanaMS** [comment]
Made the PR changes. Merging to dev.

**Pull request merged**